com.github.Cogitri.gxi.yml

```diff
@@ -0,0 +1,28 @@
+app-id: com.github.Cogitri.gxi
+runtime: org.freedesktop.Platform
+runtime-version: "18.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+command: gxi
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=home
+  - --filesystem=xdg-run/dconf
+  - --filesystem=xdg-config/dconf:ro
+  - --filesystem=xdg-config/gxi
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+  build-args:
+  - --share=network
```

Comment:
access to the network during the build is banned on flathub. you need to specify all dependencies in the manifest.

Reply:
I believe that's what the other rust apps do (https://github.com/search?q=org%3Aflathub+rust&type=Code can show you those)

Reply:
Yup, thanks, had looked at Podcasts before. Currently building...